src/Components/LoginPage/LoginPage.js

```diff
@@ -33,11 +34,13 @@ const LoginPage = () => {
   }, [currentUser])
 
   const loginUser = async () => {
-    const { error, loading, data } = await getUser({
-      variables: { id: signInData.successUserId }
-    })
-    // add error handling here first
-    setUser(data.fetchUser)
+      const { error, loading, data } = await getUser({
+        variables: { id: signInData.successUserId }
+      })
+      if(error) {
+        setErrorState(true)
+      }
```

Comment:
So I started with exactly what you have above and it actually didn't work. The only way I was able to get the error component to work with the login was to create an errorState and update state. I think it may be due to having multiple queries on one component.

Reply:
Oh, good to know!